treeherder/perf/management/commands/import_perf_data.py

```diff
@@ -73,6 +92,9 @@ def _add_series(pc, project_name, signature_hash, signature_props, verbosity):
                 value=datum['value'],
                 push_timestamp=datetime.datetime.fromtimestamp(datum['push_timestamp']))
 
+    signature.last_updated = latest_timestamp
+    signature.save()
```

Comment:
Put these two lines in the `try:` block, since we don't need to do this if inserting PerformanceDatum one-by-one